site/content/includes/compatibility-tables/nic-nap.md

```diff
@@ -0,0 +1,10 @@
+The following table shows compatibility between NGINX Ingress Controller and NGINX App Protect WAF versions:
+
+{{< bootstrap-table "table table-striped table-responsive" >}}
+| NIC Version | App Protect WAF v5 Version | Config Manager | Enforcer |
+| --- | --- | --- | --- |
+| {{< nic-version >}} | 33_5.264 | 5.6.0 | 5.6.0 |
```

Comment:
Waiting for information from NAP on what the v5 version name will be for this table.